src/file.ts

```diff
@@ -547,6 +551,7 @@ class File extends ServiceObject<File, FileMetadata> {
   name: string;
 
   generation?: number;
+  restoreToken?: string;
```

Comment:
Is there a reason to have this field as both a class level and metadata level field?

Reply:
I've noticed that in some scenarios, a refresh token is associated with a specific generation ID. To ensure consistent behavior and avoid potential issues, I've implemented the refresh token handling in a way that aligns with the generation ID concept. This approach will help maintain data integrity and security.